packages/components/src/internal/components/notifications/messaging.tsx

```diff
@@ -1,25 +1,12 @@
-import * as React from 'react';
+import React, { ReactNode } from 'react';
 
 import { getActionErrorMessage } from '../../..';
 
-import { createNotification } from './actions';
-
-export function createDeleteSuccessNotification(
-    noun: string,
-    count?: number,
-    additionalInfo?: string,
-) {
+export function getDeleteSuccessNotification(noun: string, count?: number, additionalInfo?: string): string {
```

Comment:
I think this function, and the one below should have a different naming convention. `get` implies that we are retrieving something already created, just stored in memory, while `Notification` implies it's going to return a `Notification` model or related. I think these should be renamed to `deleteSuccessMessage`, and `deleteErrorMessage`. I think we should also consider renaming `getActionErrorMessage` to `actionErrorMessage`, however that change is a bit bigger.

Reply:
Yeah, I struggled to come up with a good name for these. I like your suggestions of `deleteSuccessMessage` and `deleteErrorMessage`. I'll go with that, but likely leave `getActionErrorMessage` alone as part of this PR (I'll add a TODO).